lib/routes/password.js

```diff
@@ -140,8 +140,8 @@ module.exports = function (
             keys: isA.boolean().optional()
           },
           payload: {
-            authPW: isA.string().min(64).max(64).regex(HEX_STRING).required(),
-            wrapKb: isA.string().min(64).max(64).regex(HEX_STRING).required(),
+            authPW: validators.authPW,
+            wrapKb: validators.wrapKb,
```

Comment:
For consistency, please also apply a similar cleanup to the validators used in `lib/routes/account.js` and `lib/routes/session.js`.